factories/operative_manager.js

```diff
@@ -0,0 +1,11 @@
+import { Factory } from 'fishery'
+
+export const operativeManagerUserFactory = Factory.define(() => ({
```

Comment:
I would suggest to remove this `export`, as this can cause some problems if we accidentally import the `operativeManagerUserFactory` without the `.build()`